crates/linker/src/pe.rs

```diff
@@ -521,13 +812,21 @@ enum SectionKind {
     ReadOnlyData,
 }
 
+#[allow(dead_code)]
```

Comment:
Are any of these dead_code directives removable now?

Reply:
yes, they all are